src/networkwatcher.cpp

```diff
@@ -174,3 +175,31 @@ QString NetworkWatcher::getCurrentTransport() {
   return QString(metaEnum.valueToKey(type))
       .remove("TransportType_", Qt::CaseSensitive);
 }
+
+void NetworkWatcher::setImpl(NetworkWatcherImpl* networkWatcherImpl) {
+  logger.debug() << "setImpl";
+
+  m_impl = new InspectorNetworkWatcher(this);
```

Comment:
You already created the InspectorNetworkWatcher and passed it into this function as an argument in the Command, so i'd suggest `m_impl = netWorkWatcherImpl`